### ISSUE TITLE
What are the requirements for a "clean claim"?

### ISSUE THREAD

**Issue by mthong** [cagov/unemployment#24]
**History** 
This topic came up in our ODI-Nava sync on Mon 4/12/2020, and @nguyen-tina asked for clarification in [this slack thread](https://nava.slack.com/archives/C0118STES6P/p1586810814191700). 

**Context** 
- A "clean claim" is a UI application that does not have missing info or errors require EDD to do manual processing.
- A clean claim will get processed faster and therefore the applicant will get money sooner.  
- More clean claims also reduces the workload for EDD. 
- One of the goals of our MVP screener is to maximize the number of "clean claims" by preparing users for the application process. 

**Answer so far, from Eric:**  
1) SSN or DL# match to confirm identity
2) No immigration status issue
3) No active existing UI claim in the system
4) No employers listed other than those that have reported income in the base wage file
5) Answers yes to all attestations

**Still waiting on**
Validation from EDD of what the complete list is - assume this will come from Eric (who I can't tag in GH yet, so I just tagged @nguyen-tina.

**Comment by ehysen** [comment]
Updated list that I drafted:

1. SSA and/or DMV validation is successful without requiring further documentation
2. For non-citizens, SAVE validation is successful without requiring further documentation
3. For UI claims, all employers entered by the claimant have provided data in the base wage file for that claimant
4. Claimant does not already have a UI claim or exhausted benefits for the year (for UI)
5. Claimant answers yes to all required attestations

Per EDD: "I don’t think we have a definitive list for clean claim scenarios for either UI or PUA in a standalone document form (have them as business rules in our design docs), but you have the main items below." So I'd consider this the list.
